test_files/export/export_star_imported.js

```diff
@@ -12,18 +12,17 @@ exports.export1 = export_helper_1.export1;
 exports.export3 = export_helper_1.export3;
 exports.export5 = export_helper_1.export5;
 exports.export4 = export_helper_1.export4;
-const tsickle_forward_declare_2 = goog.forwardDeclare("test_files.export.export_helper");
-/** @typedef {!tsickle_forward_declare_2.Bar} */
+/** @typedef {!tsickle_forward_declare_1.Bar} */
 exports.Bar; // re-export typedef
-/** @typedef {!tsickle_forward_declare_2.TypeDef} */
+/** @typedef {!tsickle_forward_declare_1.TypeDef} */
```

Comment:
Why are these tests changing?

Reply:
Previously, the code eagerly emitted a duplicated `forwardDeclare` for `export_helper`, because emitting a forwardDeclare for an `export *` did not check whether the module symbol already had a forward declare. I've harmonized that code during the rewrite.

(I think that is, it's two weeks ago that I wrote this by now).